server/src/main/kotlin/org/javacs/kt/codeaction/quickfix/AddMissingImportsQuickFix.kt

```diff
@@ -0,0 +1,59 @@
+package org.javacs.kt.codeaction.quickfix
+
+import org.eclipse.lsp4j.*
+import org.eclipse.lsp4j.jsonrpc.messages.Either
+import org.jetbrains.kotlin.psi.KtFile
+import org.javacs.kt.CompiledFile
+import org.javacs.kt.LOG
+import org.javacs.kt.index.SymbolIndex
+import org.javacs.kt.index.Symbol
+import org.javacs.kt.position.offset
+import org.javacs.kt.util.toPath
+import org.javacs.kt.codeaction.quickfix.diagnosticMatch
+import org.javacs.kt.imports.getImportTextEditEntry
+
+class AddMissingImportsQuickFix: QuickFix {
+    override fun compute(file: CompiledFile, index: SymbolIndex, range: Range, diagnostics: List<Diagnostic>): List<Either<Command, CodeAction>> {
+        val uri = file.parse.toPath().toUri().toString()
+        val unresolvedReferences = getUnresolvedReferencesFromDiagnostics(diagnostics) 
+        
+        return unresolvedReferences.flatMap { diagnostic ->
+            val diagnosticRange = diagnostic.range
+            val startCursor = offset(file.content, diagnosticRange.start)
+            val endCursor = offset(file.content, diagnosticRange.end)
+            val symbolName = file.content.substring(startCursor, endCursor)
+
+            getImportAlternatives(symbolName, file.parse, index).map { (importStr, edit) ->
+                val codeAction = CodeAction()
+                codeAction.title = "import ${importStr}"
```

Comment:
For consistency with the other quick fix I would capitalize this:

```suggestion
                codeAction.title = "Import ${importStr}"
```